tests/token_integration_tests.rs

```diff
@@ -1,29 +1,172 @@
-use kora_lib::token::{TokenProgram, TokenType, TokenInterface};
-use solana_client::rpc_client::RpcClient;
-use solana_sdk::pubkey::Pubkey;
+use kora_lib::{
+    error::KoraError,
+    token::{Token2022Account, TokenInterface, TokenProgram, TokenState, TokenType},
+    transaction::validator::validate_token2022_account,
+};
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::{
+    commitment_config::CommitmentConfig,
+    pubkey::Pubkey,
+    recent_blockhash::RecentBlockhash,
+    signature::{Keypair, Signer},
+    transaction::Transaction,
+};
+use spl_associated_token_account::get_associated_token_address_with_program_id;
+use spl_token_2022;
+use std::str::FromStr;
+
+// PYUSD token mint on devnet
+const PYUSD_MINT: &str = "PYUSD6FiXyYotH6a4LJDP8mjQT7eJRJ4mkd5ZUj94UF";
```

Comment:
PYUSD is actually at CXk2AMBfi3TwaEL2468s6zP8xq9NxTXjp9gjMgzeUynM
You can use paxos' faucet to get some if you need